server/bin/build-libjxl.sh

```diff
@@ -43,7 +43,9 @@ cmake \
   -DJPEGXL_ENABLE_PLUGINS=ON \
   -DJPEGLI_LIBJPEG_LIBRARY_SOVERSION="${JPEGLI_LIBJPEG_LIBRARY_SOVERSION}" \
   -DJPEGLI_LIBJPEG_LIBRARY_VERSION="${JPEGLI_LIBJPEG_LIBRARY_VERSION}" \
+  -DLIBJPEG_TURBO_VERSION_NUMBER=2001005 \
```

Comment:
Is this actually used anywhere?

Reply:
It's just to make sure `LIBJPEG_TURBO_VERSION_NUMBER` is not an empty string.